### PR TITLE
Adds documentations on 'approaches' parameter.

### PR DESCRIPTION
**Pull request by fijemax**
Adds documentations on 'approaches' parameter.
# Issue
#2201
# PR 
#4039

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # 5.8.0
   - Changes from 5.7
+    - API:
+      - new parameter `approaches` for `route`, `table`, 'trip' and `nearest` requests.  This parameter keep waypoints on the curb side
+        (depending 'left_hand_driving' parameter into lua script). 'approaches' accepts both 'curb' and 'unrestricted' values.
+    - NodeJs Bindings
+      - new parameter `approaches` for `route`, `table`, 'trip' and `nearest` requests.
```
Comment (daniel-j-h) [inline CHANGELOG.md]: Could you use \` backticks for all highlighting here and above

Comment (fijemax) [inline CHANGELOG.md]: done

**Comment by fijemax** [comment]
I added a note to explain that.

**Comment by daniel-j-h** [comment]
Perfect thank you!

(Travis test failing is unrelated, see https://github.com/Project-OSRM/osrm-backend/issues/4110)

**Pull request merged**